services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchRulesSpec.groovy

```diff
@@ -213,6 +216,56 @@ class SwitchRulesSpec extends BaseSpecification {
         ]
     }
 
+    @Unroll
+    @Tags(Tag.NEGATIVE)
+    def "Deleting switch rules by non-existing #data.description is possible but any rules are not deleted"() {
+        given: "A switch with some flow rules installed"
+        northboundService.addFlow(flow)
+        Wrappers.wait(WAIT_OFFSET) {
+            assert northboundService.getFlowStatus(flow.id).status == FlowState.UP
+            assert northboundService.getSwitchRules(data.switch.dpId).flowEntries.size() ==
+                    defaultRules.size() + flowRulesCount
+        }
+
+        when: "Delete switch rules by non-existing #data.description"
+        def deletedRules = northboundService.deleteSwitchRules(data.switch.dpId, data.inPort, data.inVlan, data.outPort)
```

Comment:
After #1545 is merged, it'd be nice to add deletion by an invalid cookie

Reply:
Is there a possibility to delete rules by cookies?

Reply:
there definitely is

Reply:
Yes, it is. So it makes sense to add such kind of tests. Will do in the next PRs.